### ISSUE TITLE
listbox does not implement listbox a11y pattern

### ISSUE THREAD

**Issue by oberhamsi** [bitmovin/bitmovin-player-ui#607]
the [listbox](https://github.com/bitmovin/bitmovin-player-ui/blob/develop/src/ts/components/listbox.ts) fits the l[istbox pattern](https://www.w3.org/WAI/ARIA/apg/patterns/listbox/)

but does not properly implement it. we were able to subclass and monkey-patch some of the problems:

 * listbox must have role=listbox
 * children of listbox muste have role=option and a unique id
 * keyboard controls for virtual focus managment (arrow keys) and aria-activedescendant as described in the pattern
 
what we could not easily change:

 * buttons used as role=option should have aria-selected and not aria-pressed states

**Comment by jamesdavidholding** [comment]
Hi Oberhamsi

We will be closing the ‘Issues’ tab for this Repo this week. If this issue still affects you, please can I ask that you re-raise it through the Bitmovin Support Dashboard, where we will happily pick it up. 

This change improves our response times to you, as the support team is not monitoring GitHub repro.  

Many thanks,

James - Support
